pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
     "pyparsebluray>=0.1.4",
     "pydantic>=2.5.0",
     "psutil>=5.6.4",
+    "videotimestamps>=0.1.0",
```

Comment:
I would recommand to bump it to 0.2.0 to support because, from that version, it also support v1 timestamps.